ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/Constraints.kt

```diff
@@ -0,0 +1,28 @@
+package net.corda.v5.ledger.utxo
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.cipher.suite.DigestService
+
+/**
+ * Defines a constraint that determines which contract-code-containing CPK can be used with a [Contract].
+ */
+@CordaSerializable
+@DoNotImplement
+interface CpkConstraint {
```

Comment:
I don't think we need to expose cpkconstraints in C5. I think the automatic sandboxing and AMQP rules will mean that this is enforced by the platform and not user modifiable so delete.